bangazonapi/views/product.py

```diff
@@ -103,7 +103,11 @@ def create(self, request):
             new_product.image_path = data
 
         new_product.save()
```

Comment:
Good job adding the `try/except` with the `clean_fields` and `save` both inside, but we don't need the first save anymore.  Only need it once!

Reply:
I think I removed the correct save, yes?